examples/ucp_client_server.c

```diff
@@ -59,6 +61,19 @@ typedef enum {
 } send_recv_type_t;
 
 
+typedef struct data_meta {
```

Comment:
> * Unify the code of single buffer and IOV: single buffer is a special case where IOV=1. all internal data structures will be iov-oriented, and just before send/recv - decide what parameter to pass (if count==1 : contig, else: IOV)
@yosefe Do you mean that the ```data_meta``` only need deal with IOV case? Before send/recv, we'll decide use contig or IOV operation, right?

Reply:
yes